pyanaconda/modules/payloads/source/url/url.py

```diff
@@ -67,6 +67,15 @@ def for_publication(self):
         """Get the interface used to publish this source."""
         return URLSourceInterface(self)
 
+    def generate_repo_configuration(self):
+        """Generate RepoConfigurationData structure."""
+        data = RepoConfigurationData()
+
+        data.name = self.type.value
+        # TODO: Use solution https://github.com/rhinstaller/anaconda/pull/2474
```

Comment:
I think that we should just return `self.repo_configuration` in this case.

Reply:
Ahh, good point. Truth is that I don't need to fill it up when it's already filed from KS data.